orbit-2-core/src/main/java/com/babylon/orbit2/Host.kt

```diff
@@ -16,9 +16,11 @@
 
 package com.babylon.orbit2
 
+@Orbit2Dsl
```

Comment:
is this one necessary? by removing it ` .sideEffect { somethingElse(action.toString()) }` becomes allowed again, but `.sideEffect { sideEffect { ... } }` is still disallowed

Reply:
Without this you can do `orbit { orbit {} }